docusaurus.config.js

```diff
@@ -256,6 +256,12 @@ const config = {
       ],
       copyright: `Copyright © ${new Date().getFullYear()} Pants project contributors. Built with Docusaurus.`,
     },
+    algolia: {
+      appId: "QD9KY1TRVK",
+      apiKey: "633f6891a9c1a7db671ce285a5a63819",
```

Comment:
Just confirming that this key is acceptable to be public?

Reply:
Yes. This is the key used to read from the index. (The Docusaurus docs even say that it's ok to check in)